src/pure.js

```diff
@@ -60,8 +60,12 @@ function render(
   return {
     container,
     baseElement,
-    // eslint-disable-next-line no-console
-    debug: (el = baseElement) => console.log(prettyDOM(el)),
+    debug: (el = baseElement) =>
+      Array.isArray(el)
+        ? // eslint-disable-next-line no-console
+          el.map(e => console.log(prettyDOM(e)))
```

Comment:
```suggestion
          el.forEach(e => console.log(prettyDOM(e)))
```

We don't want to return an array with undefined values from `debug` :)

Reply:
yeah, you're right :man_facepalming: Thank you Kent! I just applied those changes.